lib/connection_controller.go

```diff
@@ -5,12 +5,23 @@ import (
 	"github.com/btcsuite/btcd/addrmgr"
 	"github.com/btcsuite/btcd/wire"
 	"github.com/deso-protocol/core/bls"
+	"github.com/deso-protocol/core/collections"
 	"github.com/golang/glog"
 	"github.com/pkg/errors"
 	"net"
 	"strconv"
+	"sync"
+	"time"
 )
 
+type GetActiveValidatorsFunc func() *collections.ConcurrentMap[bls.SerializedPublicKey, *ValidatorEntry]
+
+var GetActiveValidatorImpl GetActiveValidatorsFunc = BasicGetActiveValidators
```

Comment:
How will GetActiveValidatorImpl be used? Seems like the intention is to globally reassign from somewhere else in the code?

Reply:
Fixed in #945